amxmodx/CLang.h

```diff
@@ -118,7 +113,7 @@ class CLangMngr
 	
 	LangVec m_Languages;
 
-	CVector<md5Pair *> FileList;
+	StringHashMap<time_t> FileList;
 	CVector<String *> KeyList;
 	THash<String, keytbl_val> KeyTable;
```

Comment:
You should try merging KeyList/KeyTable into a single HashTable from AMTL, which will mean tinyhash class will no longer be needed.